src/ios/LocalyticsPlugin.m

```diff
@@ -1191,13 +1222,27 @@ - (void)removeMessagingListener:(CDVInvokedUrlCommand *)command {
 - (void)setLocationMonitoringEnabled:(CDVInvokedUrlCommand *)command {
     [self logInput:@"setLocationMonitoringEnabled" withCommand:command];
     NSNumber *enabled = [command argumentAtIndex:0];
+    NSNumber *persist = [command argumentAtIndex:1];
```

Comment:
Not sure I understand?

Reply:
Like react, Cordova apps can be reloaded on an upgrade. when that occurs any additional argument added to a method should support an undefined value for the additional parameters and the behavior should be the same as the one without the argument.

The notion of live reload of an app exists so that the app can be loaded quickly and a start from scratch is not necessary to reduce app launch times. 
-- I learnt this from our friends from Bloomberg. I didn't know this occurred when I reviewed the first release of Cordova / React.

Reply:
I don't think this is a concern - this does handle undefined.

Our code right after which handles `nil`:
```
if ([persist isKindOfClass:[NSNumber class]]) {
  [Localytics persistLocationMonitoring:persist];
}
```
And cordova's doc on `argumentAtIndex`:
>Returns the argument at the given index. If index >= the number of arguments, returns nil. If the argument at the given index is NSNull, returns nil.